openfisca_core/simulations/simulation.py

```diff
@@ -96,7 +96,8 @@ def calculate(self, variable_name: str, period):
         """Calculate ``variable_name`` for ``period``."""
 
         if period is not None and not isinstance(period, Period):
-            period = periods.period(period)
+            ##period = periods.period(period)
+            period = Period(period)
```

Comment:
This is a breaking change (the badly named `periods.period` function is actually a builder function.